auth/ReceiverKeyStateMachine.m

```diff
@@ -208,6 +211,7 @@ function purge_expired_keys(obj, current_time, level)
     methods (Static, Access = protected)
```

Comment:
We could delete this and you could now just call the truncated_sha inside of hashing wrappers that I wrote

Reply:
excellent idea when you commit your change...I will ask you to update these classes...don't forget.